miio/miot_device.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from .device import Device
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class MiotDevice(Device):
+    """Main class representing a MIoT device."""
+
+    def __init__(
+        self,
+        mapping: dict,
```

Comment:
Who not simply define the mapping as a class variable to avoid passing it around and polluting the ctor even more? :-)

Reply:
I suggest not changing this here since it's IMO consistent with other stuff and I'm not sure about tests structure then..

Reply:
Ok, that can be cleaned later on if wanted :+1: